includes/library/zencart/platform/src/Paginator/adapters/QueryFactory.php

```diff
@@ -17,15 +17,16 @@
  */
 class QueryFactory extends AbstractAdapter implements AdapterInterface
 {
+
     /**
-     * getResultList method
-     *
-     * @return array|mixed
+     * @param $data
+     * @param array $params
+     * @return array
      */
-    public function getResultList($data, $params)
+    public function getResultList($data, array $params)
     {
         $limit = $params['currentItem'] - 1 . ',' . $params['itemsPerPage'];
-        $results = $data['dbConn']->execute($data['sqlQueries']['main'], $limit);
+        $results = $data['dbConn']->execute($data['mainSql'], $limit);
```

Comment:
`['sqlQueries']['main']` and `['sqlQueries']['count']` are also used in `zcPaginatorAdapterQuery` but that file is missing from this commit.

So, given that this commit is changing the signature, I suspect the refactoring needs to be applied to `class.zcPaginatorAdapterQuery.php` as part of this PR too.